test/test_basic_stubbing.py

```diff
@@ -109,3 +110,15 @@ def test_default_count_for_stubs_is_unlimited(self):
 
         result = requests.get(MOCK_SERVER_URL + "/path")
         self.assertEqual(result.status_code, 200)
+
+    def test_time_to_live_stubbing(self):
+        self.client.stub(
+            request(),
+            response(),
+            time_to_live=time_to_live(seconds(1))
```

Comment:
I think the api would be better with

`time_to_live=1  # which would default to seconds, like the delay`